pkg/render/node.go

```diff
@@ -1736,14 +1736,15 @@ func (c *nodeComponent) nodeLivenessReadinessProbes() (*corev1.Probe, *corev1.Pr
 				Port: livenessPort,
 			},
 		},
+		PeriodSeconds:  60,
 		TimeoutSeconds: 10,
 	}
 	rp := &corev1.Probe{
 		ProbeHandler: corev1.ProbeHandler{Exec: &corev1.ExecAction{Command: readinessCmd}},
 		// Set the TimeoutSeconds greater than the default of 1 to allow additional time on loaded nodes.
 		// This timeout should be less than the PeriodSeconds.
+		PeriodSeconds:  30,
 		TimeoutSeconds: 5,
```

Comment:
For node I'd go higher; 10s timeout and maybe a 5 minute initial delay.  On large clusters; restarting within 5 mins is more likely to cause a cyclic restart than fix the problem.

Reply:
> maybe a 5 minute initial delay

This is not included in this PR. We may want to properly design and implement this as suggested by @tmjd.